app/controllers/api_controller.rb

```diff
@@ -22,6 +22,7 @@ class ApiController < ApplicationController
   rescue_from Faraday::ConnectionFailed, Faraday::TimeoutError, with: :render_connection_error
   rescue_from ActiveModel::ValidationError, with: :render_validation_error
   rescue_from IncludeParamsValidator::ValidationError, with: :render_include_validation_error
+  rescue_from Idempotency::ConflictError, with: :render_conflict_error
```

Comment:
You could move this line and the corresponding method below to the Idempotentable concern.